Makefile

```diff
@@ -60,19 +59,18 @@ config.json : code/config.ss
 
 # documentation
 docs/%.md : code/md.ss code/docs/%.ss
-	$(call exercise, "(put-md '$(@F:.md=))")
+	$(call exercise, "(put-doc '$(@F:.md=))")
 
-$(readme-splice) : docs/TESTS.md
-	cp $< $@
+$(readme-splice) : $(track-documentation)
+	cp docs/TESTS.md $@
```

Comment:
Wasn't building properly since it couldn't see docs/TESTS.md as a rule. Running a second make would actually get the file copied. This fixes that